packages/core/src/utils.ts

```diff
@@ -40,3 +40,22 @@ export type AnyCase<T extends string> = Uppercase<T> | Lowercase<T>;
 export function isKey<T extends {} | []>(key: string | number | symbol, of: T): key is keyof T {
   return key in of;
 }
+
+/**
+ * Function to encode a number as hex in format low to high values
+ * @param {[Number]} input 
+ * @param {[Number]} length 
+ * @returns {[String]} hex value low-high order
+ */
+export function intLowHigh(input: number, length: number = 1): String {
+  let ret = ''
+  for (let i = 0; i < length; i++) {
+      let value = (input % 256).toString(16).toUpperCase();
+      if (value.length % 2 !== 0) {
+          value = `0${value}`;
+      }
```

Comment:
```ts
if (value.length > length) {
      break
}
```

Reply:
I am not sure why this check should be added? It should always be a hex couple as 0x00. The parameter length means the length of the hex couples 0x00 0x01 ... I have added this hint to the parameter. 0x0 is not intended as valid result.

Reply:
OH, You are right.